tools/deployment/package-helm/templates/service-account-role.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: "rbac.authorization.k8s.io/v1"
 kind: "Role"
 metadata:
-  name: {{ include "clp.fullname" . }}-job-watcher
+  name: {{ include "clp.fullname" . }}-service-account
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)